src/components/ErrorBoundary/index.tsx

```diff
@@ -126,7 +126,7 @@ function getRelevantState(): null | keyof AppState {
   if (!path.startsWith('#/')) {
     return null
   }
-  const pieces = path.substring(2).split(/[\/\\?]/)
+  const pieces = path.substring(2).split(/[/\\?]/)
```

Comment:
Is this change intentional?

Reply:
yeah, linter caught an unnecessary escape